jenkins-plugin-cli.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-java -jar /usr/lib/jenkins-plugin-manager.jar "$@"
+exec /bin/bash -c "java $JAVA_OPTS -jar /usr/lib/jenkins-plugin-manager.jar $*"
```

Comment:
Before this change, this script worked on computers that did not install `bash` in `/bin/bash`.  With the change, it now requires that bash must be installed at `/bin/bash`.  Since this is only in the Docker image, that seems OK since it is a safe assumption that all Linux Docker images have /bin/bash available.  I confirmed that the Jenkins alpine image includes /bin/bash.

If that is the requirement, won't we need to confirm that all the Docker images have bash installed as `/bin/bash`? 

If that is the requirement, shouldn't we switch the script entry from `#!/usr/bin/env bash` to #!/bin/bash`?
```suggestion
#!/bin/bash

exec /bin/bash -c "java $JAVA_OPTS -jar /usr/lib/jenkins-plugin-manager.jar $*"
```

Reply:
Sounds reasonable

Reply:
@slide @MarkEWaite All checks have passed. Can it be merged now?